core/domain/question_domain_test.py

```diff
@@ -328,7 +328,7 @@ def test_strict_validation_for_answer_groups(self):
                     'rule_type': 'Contains'
                 }],
                 'training_data': [],
-                'tagged_misconception_id': None
+                'tagged_skill_misconception_id': None
```

Comment:
Shouldn't the migration function have a test?

Reply:
Done.

Reply:
Wait where is the test? Don't we need a test in question_domain_test as well?

Reply:
It's being tested in question_services_test, in order to keep it consistent with skills.